SweaterWeather/js/script.js

```diff
@@ -0,0 +1,116 @@
+var geoButton = document.getElementById('geo');
+var notifyButton = document.getElementById('notify');
+
+// Check onload status of geolocation permission.   
+navigator.permissions.query({name:'geolocation'}).then(function(result){ 
+ if (result.state == 'denied') {
+  geoButton.setAttribute('icon','communication:location-off');
+  geoButton.style.color = 'grey';
+ }
+ else if (result.state == 'granted') {
+  geoButton.setAttribute('icon','communication:location-on');
+  geoButton.style.color = 'rgb(62,81,181)';
+ }
+ else { // Prompt
+  geoButton.setAttribute('icon','communication:location-on');
+  geoButton.style.color = 'grey';
+ } 
+});
+
+// Check onload status of notifications permission.
+navigator.permissions.query({name:'notifications'}).then(function(result){ 
+ if (result.state == 'denied') {
+  notifyButton.disabled = true;
+  notifyButton.checked = false;   
+  notifyButton.style.color = 'grey';
+ }
+ else if (result.state == 'granted') {
+  notifyButton.disabled = false;
+  notifyButton.checked = true; 
+  notifyButton.style.color = 'rgb(62,81,181)';   
+ }
+ else {  // Prompt
+  notifyButton.disabled = false;
+  notifyButton.checked = false; 
+  notifyButton.style.color = 'grey';
+ } 
+});
+
+// Listen for when location button is clicked.  
+document.addEventListener('DOMContentLoaded', function(event){
+ geoButton.addEventListener('click', function(){
+  navigator.permissions.query({name:'geolocation'}).then(function(result){
+   if (result.state == 'granted'){
```

Comment:
same here, the if-statement is indented 3 spaces, it looks like? also, a space: `'granted') {`